syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Transaction {
    int32 sid = 1;
    float amount = 2;
    google.protobuf.Timestamp dt = 3;
    float price = 4;
    string order_id = 5;
}